x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/whitelist/InternalSqlScriptUtils.java

```diff
@@ -235,7 +236,7 @@ public static Double asin(Number value) {
     public static Double atan(Number value) {
         return MathOperation.ATAN.apply(value);
     }
-    
+
```

Comment:
?

Reply:
Thought is ok to fix the formatting since it's only touching empty lines.